docs/examples/alpine/templates/alpine-pod.yaml

```diff
@@ -3,8 +3,8 @@ kind: Pod
 metadata:
   name: {{.Release.Name}}-{{.Chart.Name}}
```

Comment:
The linter does not enforce `| quote` in this case, since it makes an invalid yaml file `name: "foo"-"bar"`